src/main.rs

```diff
@@ -21,6 +27,64 @@ use net::{
 use stm32h7xx_hal::hal::digital::v2::OutputPin;
 use systick_monotonic::*;
 
+#[derive(Copy, Clone, Debug, Miniconf, Format)]
+pub struct OutputSettings {
+    /// En-/Disables the TEC driver.
+    ///
+    /// # Path
+    /// `output_settings/<n>/enable`
+    ///
+    /// * <n> specifies which channel to configure. <n> := [0, 1, 2, 3]
+    ///
+    /// # Value
+    /// true to enable, false to disable.
+    pub enable: bool,
+
+    /// Maximum negative TEC current in ampere.
+    ///
+    /// # Path
+    /// `output_settings/<n>/max_i_neg
+    ///
+    /// * <n> specifies which channel to configure. <n> := [0, 1, 2, 3]
+    ///
+    /// # Value
+    /// 0.0 to 3.0
```

Comment:
Why is it positive then? C.f. https://github.com/quartiq/thermostat-eem/pull/18/files#r824153422